docs/csharp/quick-starts/hello-world.yml

```diff
@@ -78,6 +78,14 @@ items:
 
     Click **Run** again to see the results.
 
+    As we progress, creating strings to print using `+` becomes cumbersome. There's an easier way: Put a `$` before the opening " and you can place variables inside `{` and `}`. Try it:
```

Comment:
maybe "variables or expressions"?

Reply:
Here is the description we used for the In-Browser Tutorial on the marketing site. It's been thru a few iterations, a copy edit, and did well in the user study. It may not be right for here, but thought I'd share in case it is useful 😄.

> String interpolation lets you piece together strings in a more concise and readable way.
>
> If you add a $ before the opening quotes of the string, you can then include string values, like the name variable, inside the string in curly brackets.

Reply:
I took Rowan's text, but changed "value" to "expression".